### ISSUE TITLE
Audio gets muted

### ISSUE THREAD

**Issue by VirtualReign** [zWolfrost/Better-Volume-Booster#6]
Hi.

I'd only just started using the add-on, so I may be missing something, but I've run into the issue of audio being muted when opening new pages with the add-on activated.

So far I'd only tested this on twitch.tv and imdb.com.

The add-on seems to work well initially, but upon returning to the site the audio is muted.

Deactivating the add-on and refreshing the page fixes the issue, and reactivating the add-on after restores the volume setting for that page, however upon closing the page and reopening it, the audio is muted again.

**Comment by zWolfrost** [comment]
Hi, does this still happen when you give the extension full access to all websites in its permission tab? (Assuming you didn't already). Nevertheless, i will be able to properly debug this issue in a few days, so please be patient :)

**Comment by VirtualReign** [comment]
Yes, the extension already has that permission enabled.

**Comment by zWolfrost** [comment]
I'm sorry but i was not able to reproduce this issue. The extension works like expected.

Could you try again in a new private window / clean firefox profile (to be sure that this problem is not caused because of conflicts with other add-ons or bad firefox preferences)?

**Comment by VirtualReign** [comment]
The issue still occurred for me in private windows. I'd tried disabling some of the extensions that provide similar sound-related functions and the issue persisted.

I currently don't have a separate profile set up, but I might get around to setting one up and gradually adding the extensions I have to see if that's the cause when I have more time on my hands.

**Comment by zWolfrost** [comment]
Oddly enough, a few months ago someone had a problem which sounds very similar like yours (https://github.com/zWolfrost/Better-Volume-Booster/issues/5#issuecomment-1962198018) on kick.com, and he fixed it by updating his GPU drivers.

I frankly don't believe that's the problem, as I can't think of any way a GPU can mess with an extension like this, but I felt like that was worth pointing out.

Just to be sure, are you using the latest version of the add-on? (1.10.2)

**Comment by KatayR** [comment]
I'm experiencing the same problem. When I uninstall the extension and reload the page, it gets fixed.

It only happens on facebook and instagram tho.

**Comment by zWolfrost** [comment]
Your updated version of the manifest.json does not really solve the problem; It just blacklists the websites. I would prefer not to do that, as they work fine for me and I'm sure for other people as well. At least I hope so.

**Comment by KatayR** [comment]
> Your updated version of the manifest.json does not really solve the problem; It just blacklists the websites. I would prefer not to do that, as they work fine for me and I'm sure for other people as well. At least I hope so.

Yeah I know it doesn't solve the root of the problem. I just didn't feel like hunting bugs rn.

**Comment by zWolfrost** [comment]
> Yeah I know it doesn't solve the root of the problem. I just didn't feel like hunting bugs rn.

Don't worry, it's fine as long as your workaround works for you. However I'm leaving this issue open until I can reproduce and eventually fix this bug.

**Comment by pkiing** [comment]
I'm facing this exact issue. Audio is completely muted when using this addon. Have tried the browser with only this addon enabled and still having the problem

**Comment by Miyazono1** [comment]
i'm actually having 2 issues because of this extension, one of them is that described here already. it used to work fine, and now it doesn't. i only use it for a couple of sites, so i didn't really take note of when exactly it stopped working.

the other issue i have is that with this extension enabled, certain types of videos will not load, and give the error "no video with supported format and mime type found." disabling this extension fixes the issue.

**Comment by zWolfrost** [comment]
@Miyazono1 it's weird that it works again for you as I didn't really change much related to the audio boosting... Well, I'm not complaining.
Anyways, you can blacklist a website by right-clicking the extension while being on the page you want to blacklist. You should see an option in the context menu there.

![Image](https://github.com/user-attachments/assets/1e34f93e-3f2e-4cfe-b2a0-d810c59fdb2d)

**Comment by Miyazono1** [comment]
i did totally uninstall the other version before installing the test version, so maybe it just needed a refresh, i have no idea lmao. also ahh ok i see it. maybe also add it to the actual menu when you left click the extension button too, just for easier access? or maybe also an actual domain list you can manually add links to so you can just add them yourself in bulk without having to go to the site?

**Comment by zWolfrost** [comment]
@Miyazono1 I personally an editable domain list be quite cumbersome and inconvenient to make & use. Things are probably better this way. Although, I will definitely make it more clear in the settings page that those options do exist.

**Comment by zWolfrost** [comment]
Apparently the autoplay block was messing with the extension's ability to boost audio. I'm not sure if it's the only reason why this would happen, but it definitely fixed it for me when testing this on a new profile.

If this was previously disabled, allowing it will probably fix the issue. Let me know if it works.

<img width="548" height="270" alt="Image" src="https://github.com/user-attachments/assets/dfd71d8c-3b22-4489-b8e3-b244b88fb195" />
